packages/react-composites/src/composites/CallComposite/adapter/CallAdapter.ts

```diff
@@ -46,7 +48,9 @@ export const END_CALL_PAGES: CallCompositePage[] = [
   'accessDeniedTeamsMeeting',
   'joinCallFailedDueToNoNetwork',
   'leftCall',
-  'removedFromCall'
```

Comment:
This is an `@private` symbol. Add the new values without conditional compilation. It might help you reduce some conditional compilation elsewhere in the logic as well.